docs/management/managing-licenses.asciidoc

```diff
@@ -21,9 +21,9 @@ and more. For example, the following pages are added to the side navigation:
 The **Management** page gains options related to configuring security, managing 
 {ls} pipelines, and using {watcher}. 
 
-At the end of the trial period, your license reverts
-to a Basic license. To continue using the Platinum features,
-purchase a subscription and update your license.
+At the end of the trial period, the Platinum features operate in a 
+{stack-ov}/license-expiration.html[degraded mode]. You can purchase a 
+subscription, revert to a Basic license, or extend the trial.
```

Comment:
Minor nit, but the text here should probably align with https://github.com/elastic/stack-docs/pull/73/files#diff-d7f42e71d0b5f3d1aff550108d0f9029R20  ? I'd love to find ways to reduce the duplication, but it's not overwhelming here.